internal/entities/command_tags.go

```diff
@@ -19,14 +19,25 @@ var (
 	entityTags []string
 )
 
-var entitiesDescribeTags = &cobra.Command{
-	Use:   "describe-tags",
+var cmdTags = &cobra.Command{
+	Use:   "tags",
+	Short: "Manage tags on New Relic entities",
+	Long: `Manage entity tags
+
+The tag command allows users to manage the ags that are applied on the specified
+entity. Use --help for more information.
+`,
+	Example: "newrelic entity tags describe --guid <guid>",
+}
+
+var cmdTagsDescribe = &cobra.Command{
+	Use:   "describe",
```

Comment:
Here's an opportunity to sub something else for `describe` if we want (`list`?)

Reply:
Good call, I dislike describe.  I was thinking `show`, `get`, `list`